lti_consumer/tests/unit/test_models.py

```diff
@@ -99,11 +101,12 @@ def test_lti_consumer_ags_enabled(self):
                 'https://purl.imsglobal.org/spec/lti-ags/claim/endpoint':
                 {
                     'scope': [
-                        'https://purl.imsglobal.org/spec/lti-ags/scope/lineitem',
+                        'https://purl.imsglobal.org/spec/lti-ags/scope/lineitem.readonly',
                         'https://purl.imsglobal.org/spec/lti-ags/scope/result.readonly',
                         'https://purl.imsglobal.org/spec/lti-ags/scope/score',
                     ],
-                    'lineitems': 'https://example.com/api/lti_consumer/v1/lti/2/lti-ags'
+                    'lineitems': 'https://example.com/api/lti_consumer/v1/lti/2/lti-ags',
+                    'lineitem': 'https://example.com/api/lti_consumer/v1/lti/2/lti-ags/1'
```

Comment:
(Throughout:) please add a trailing comma after the last item to avoid excess diffs like we see here.

Reply:
I've added trailing comma to all related lists to this PR. I didn't go through other lists as that will make a lot of unrelated diff in this PR.

Reply:
Thanks :)